h/static/scripts/controllers.coffee

```diff
@@ -118,6 +121,9 @@ class AppController
     $scope.clearSelection = ->
       $scope.search.query = ''
       annotationUI.clearSelectedAnnotations()
+      crossframe.notify
```

Comment:
Something about this seems wrong to me. Specifically, the `annotationUI` service is supposed to be the canonical source of the annotation UI state. That means that we shouldn't have to call two different methods to call in order to clear the selection in both frames.

Perhaps the right thing to do is to move the `annotationUI` mutator method into a listener in `annotationUISync`, so that only the crossframe call is here.